src/helix-ui/elements/_hx-toast.less

```diff
@@ -0,0 +1,119 @@
+@import "./_hx-element";
+@import "elements/hx-icon";
+@import (reference) 'components/box';
+@import (reference) 'components/buttons';
+
+#wrapper {
+  &:extend(.hxBox-sm);
+  background-color: @gray-0;
+  box-shadow: 0px 3px 3px 0 @shadow-color;
+  color: @gray-900;
+  display: flex;
+  min-height: 3.5rem; //56px
+  position: relative;
+  width: 22rem; //352px
+
+  &::before,
+  &::after {
+    content: '';
+    display: block;
+    height: 100%;
+    left: 0;
+    pointer-events: none;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+  
+  &::before {
+    border: 1px solid @gray-300;
+  }
+
+  &::after {
+    border-left: 8px solid currentColor;
+  }
+}
+
+#icon-wrapper {
+  align-items: center;
+  display: flex;
+
+  hx-icon {
+    height: 2rem;
+    width: 2rem;
+  }
+}
+
+#content {
+  flex-direction: column;
+  flex-grow: 1;
+  flex-wrap: wrap;
+  margin: 0 0.75rem;
+  text-align: right;
+  word-wrap: break-word;
+
+  p {
+    font-size: 0.875rem;
+    font-weight: 400;
+    margin: 0;
+    text-align: left;
+  }
+}
+
+#cta {
+  &:extend(.hxBtn);
```

Comment:
Double-check that both extends are needed.

Reply:
needed -- if taken out breaks the btn link styles, do we wanna look further into this @CITguy ?

Reply:
leave it for now